### PR TITLE
doc: update sd-jwt and sd-jwt-vc link

### PR DESCRIPTION
**Pull request by lukasjhan**
Hi guys, I noticed a outdated URL of SD-JWT specification. So I updated the link to the latest version which is 07.
And I added SD-JWT-VC spec too :)

Thank you.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -181,7 +181,8 @@ Resources that are not about specific concepts such as meetups, newsletters, etc
 ## 📄 Technical Specifications
 - [W3C Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) - Specification of verifiable credentials (VCs).
 - [W3C Verifiable Credentials Implementation Guidelines](https://www.w3.org/TR/vc-imp-guide/) - Implementation guidelines for verifiable credentials.
-- [SD-JWT](https://www.ietf.org/archive/id/draft-fett-oauth-selective-disclosure-jwt-02.html) - Specification for selective disclosure for JWT based verifiable credentials.
+- [SD-JWT](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-08.html) - Specification for selective disclosure for JWT based verifiable credentials.
```
Comment (berendsliedrecht) [inline readme.md]:
```suggestion
- [SD-JWT](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/) - Specification for selective disclosure for JWT based verifiable credentials.
```

File: readme.md [hunk 2]
```diff
@@ -181,7 +181,8 @@ Resources that are not about specific concepts such as meetups, newsletters, etc
...
+- [SD-JWT-VC](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-03.html) - Specification for using SD-JWT as verifiable credentials.
```
Comment (berendsliedrecht) [inline readme.md]:
```suggestion
- [SD-JWT-VC](https://datatracker.ietf.org/doc/draft-ietf-oauth-sd-jwt-vc/) - Specification for using SD-JWT as verifiable credentials.
```

**Comment by berendsliedrecht** [comment]
I think it is better to link to the start page and let the user go to the latest version.

**Comment by lukasjhan** [comment]
> I think it is better to link to the start page and let the user go to the latest version.

That's good Idea. I'll commit your suggestion

**Review by berendsliedrecht — APPROVED** [review]

**Pull request merged**